core/clipboard/clipboard.view.js

```diff
@@ -0,0 +1,208 @@
+import {View} from '../view';
+import {Command} from '../command';
+import {getType, isUndefined} from '../utility';
+import {SelectionCommandManager} from './../selection/selection.command.manager';
+import {get, getFactory} from '../services/value';
+
+export class ClipboardView extends View {
+	constructor(model, commandManager) {
+		super(model);
+
+		const selectionCommandManager = new SelectionCommandManager(model, commandManager);
+		const shortcut = model.action().shortcut;
+		const commands = this.commands;
+
+		const view = document.querySelector('.q-grid-view');
```

Comment:
this is depricated in core, make a directive for that, or do it in view-core component